I don't understand the error message. Any idea how to fix?
```
astropy_bot/autolabel.py:74: in autolabel
    al_config = upstream_repo.get_config_value("autolabel", {})
../../../virtualenv/python3.8.0/lib/python3.8/site-packages/baldrick/github/github_api.py:156: in get_config_value
    cfg = self.get_repo_config(branch=branch)
../../../virtualenv/python3.8.0/lib/python3.8/site-packages/baldrick/github/github_api.py:123: in get_repo_config
    file_content = self.get_file_contents(path_to_file, branch=branch)
../../../virtualenv/python3.8.0/lib/python3.8/site-packages/baldrick/github/github_api.py:282: in get_file_contents
    return super().get_file_contents(path_to_file, branch=branch)
../../../virtualenv/python3.8.0/lib/python3.8/site-packages/baldrick/github/github_api.py:86: in get_file_contents
    response = requests.get(url_file, params=data, headers=self._headers)
../../../virtualenv/python3.8.0/lib/python3.8/site-packages/baldrick/github/github_api.py:68: in _headers
    return github_request_headers(self.installation)
../../../virtualenv/python3.8.0/lib/python3.8/site-packages/baldrick/github/github_auth.py:106: in github_request_headers
    token = get_installation_token(installation)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
installation = <MagicMock name='mock.installation' id='140361860340368'>
    def get_installation_token(installation):
        """
        Get access token for installation
        """
    
        now = datetime.datetime.now().timestamp()
    
        if installation_token_expiry[installation] is None or now + 60 > installation_token_expiry[installation]:
    
            # FIXME: if .netrc file is present, Authorization header will get
            # overwritten, so need to figure out how to ignore that file.
            if netrc_exists():
                raise Exception("Authentication does not work properly if a ~/.netrc "
                                "file exists. Rename that file temporarily and try again.")
    
            headers = {}
            headers['Authorization'] = 'Bearer {0}'.format(get_json_web_token())
            headers['Accept'] = 'application/vnd.github.machine-man-preview+json'
    
            url = 'https://api.github.com/app/installations/{0}/access_tokens'.format(installation)
    
            req = requests.post(url, headers=headers)
            resp = req.json()
    
            if not req.ok:
                if 'message' in resp:
>                   raise Exception(resp['message'])
E                   Exception: A JSON web token could not be decoded
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ os:
   - linux
 
 python:
-  - "3.6"
+  - "3.8"
 
 env:
   global:
@@ -16,7 +16,7 @@ env:
 install:
   - pip install -r requirements.txt
   - pip install -r requirements_test.txt
-  
+
 script:
   - pytest -rx --flake8 --cov astropy_bot astropy_bot
   - python run.py --skip-run

--- a/astropy_bot/tests/test_autolabel.py
+++ b/astropy_bot/tests/test_autolabel.py
@@ -30,6 +30,7 @@ class AutolabelBase:
 
         self.repo_handler = MagicMock()
         self.repo_handler.get_all_labels = self.get_all_labels
+        self.repo_handler.get_config_value = self.get_config_value
 
     # Patched methods:
     def get_modified_files(self):
@@ -41,6 +42,9 @@ class AutolabelBase:
     def set_labels(self, labels):
         self.labels = labels
 
+    def get_config_value(self, *args, **kwargs):
+        return {}
+
 
 class TestAutolabel1(AutolabelBase):
     files = ['CHANGES.rst', 'astropy/io/fits/convenience.py',
@@ -51,8 +55,9 @@ class TestAutolabel1(AutolabelBase):
 
     def test_autolabel_subpackages(self, app):
         with app.app_context():
-            with patch.object(RepoHandler, 'get_all_labels') as p:
+            with patch.object(RepoHandler, 'get_all_labels') as p, patch.object(RepoHandler, 'get_config_value') as p2:
                 p.side_effect = self.get_all_labels
+                p2.side_effect = self.get_config_value
                 autolabel(self.pr_handler, self.repo_handler)
 
         # make sure 'Docs' stays in there:
@@ -70,8 +75,9 @@ class TestAutolabel2(AutolabelBase):
 
     def test_autolabel_subpackages(self, app):
         with app.app_context():
-            with patch.object(RepoHandler, 'get_all_labels') as p:
+            with patch.object(RepoHandler, 'get_all_labels') as p, patch.object(RepoHandler, 'get_config_value') as p2:
                 p.side_effect = self.get_all_labels
+                p2.side_effect = self.get_config_value
                 autolabel(self.pr_handler, self.repo_handler)
 
         # make sure 'Docs' stays in there:

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,9 @@
 [tool.astropy-bot]
 
+[tool.astropy-bot.autolabel]
+
+enabled = true
+
 [tool.astropy-bot.milestones]
 
 enabled = true

